src/components/crownConfig.vue

```diff
@@ -204,7 +182,25 @@ export default {
 
       this.$emit('save-state');
     },
+    repositionCrown() {
+      const shapeView = this.shape.findView(this.paper);
+
+      if (!shapeView) {
+        return;
+      }
+
+      const { x, y, width } = shapeView.getBBox();
+
+      this.style = {
+        top: `${y - 45}px`,
+        left: `${x + width - 20}px`,
+        cursor: 'pointer',
+      };
+    },
     setUpCrownConfig() {
+      this.paper.on('render:done scale:changed translate:changed', this.repositionCrown);
```

Comment:
I triggered custom events on the paper to handle certain changes that required repositioning the crown. I found this allowed me to get an actual width from `getBBox`.